easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -63,7 +63,7 @@
 from easybuild.framework.easyconfig.licenses import EASYCONFIG_LICENSES_DICT, License
 from easybuild.framework.easyconfig.parser import DEPRECATED_PARAMETERS, REPLACED_PARAMETERS
 from easybuild.framework.easyconfig.parser import EasyConfigParser, fetch_parameters_from_easyconfig
-from easybuild.framework.easyconfig.templates import template_constant_dict
+from easybuild.framework.easyconfig.templates import template_constant_dict, TEMPLATE_CONSTANTS
```

Comment:
put constant first in import line (to keep import lines consistent)

```python
from easybuild.framework.easyconfig.templates import TEMPLATE_CONSTANTS, template_constant_dict
```

did you check how `template_constant_dict` relates to `TEMPLATE_CONSTANTS`? sure you can't just use `template_constant_dict` (the names seems to suggest you can)